botorch/models/gp_regression.py

```diff
@@ -97,7 +97,10 @@ def __init__(
         if outcome_transform is not None:
             train_Y, _ = outcome_transform(train_Y)
         self._validate_tensor_args(X=transformed_X, Y=train_Y)
-        validate_input_scaling(train_X=transformed_X, train_Y=train_Y)
+        ignore_X_dims = getattr(self, "cat_dims", None)
```

Comment:
Let's make this more generic so it can be used by other models as well? We could do sth like `_ignore_X_dims_scaling`?

Reply:
Sounds good!